### PR TITLE
Enabled Clear, Download file & Print button

### PR DESCRIPTION
**Pull request by Bsharma4**
- enabled clear button functionality
- enabled download button functionality
- enabled print button functionality
**my Git was not in proper sync so had to reupload the file
@MarshallOkafor @SUDANGOKUL @saikoneru7 kindly review and approve.
Your feedback will be appreciated

### CODE REVIEW

**Comment by saikoneru7** [comment]
Great job bro! However I'm working on Irish spell checker code it will completed by tomorrow maximum.

File: index.html [hunk 1]
```diff
@@ -1,185 +1,221 @@
...
-				
-							
-                        </script>
-
-        </body>
-
-        <!--End of page -->
-
-</html>
+<!DOCTYPE html/>
```
Comment (MarshallOkafor) [inline index.html:1]: Great one Harry! Well done for fixing those buttons. This is a good addon to our software.

I noticed that you reuploaded the entire HTML file. Well, try and always fix Git to be able to sync with the remote repo so you do not have to reupload files.

File: index.html [hunk 2]
```diff
@@ -1,185 +1,221 @@
...
+                                                },
+                                                onInput(event) {
+                                                        const value = event.target.innerText;
+                                                        this.event = event
+                                                        this.text = value;
+                                                },
+                                        },
+                                        watch: {}  
+                                })
+                                // download function
```
Comment (MarshallOkafor) [inline index.html:175]: Great one Harry! This works for our download button. With this users can download their file from our software!

File: index.html [hunk 3]
```diff
@@ -1,185 +1,221 @@
...
+                                                newLink.href = window.URL.createObjectURL(textToBLOB);
+                                                newLink.style.display = "none";
+                                                document.body.appendChild(newLink);
+                                        }
+
+                                        newLink.click();
+                                }
+
+                                // printbutton function
+                                function printText()
```
Comment (MarshallOkafor) [inline index.html:201]: Well done Harry! I can see that this code here is to handle print!. This is awesome.

**Review by MarshallOkafor — APPROVED** [review]
Well done Harry! Now we have the Print and Download buttons working.

This awesome and a good addon to our software. What about the "Clear" button, I thought that was implemented too? I could not find it here.

File: index.html [hunk 4]
```diff
@@ -1,185 +1,221 @@
...
+                        <div class="container px-4 pt-4">
+                                <div class="d-flex align-items-center p-3 my-3 text-white bg-blue rounded shadow-sm">
+                                        <div class="lh-1 p-3">
+                                                <h1 class="h3 mb-0 text-white lh-1">G4 Spell Checker Machine</h1>
+                                        </div>
+                                </div>
+                        <!-- The menu to use to manipulate the HTML page -->
+                        <div class="row g-3 py-3 pt-5 row-cols-1 row-cols-lg-2">
+                                <div class="col">
+                                        <button id='clr' type="button" class="btn  btn-sm btn-outline-primary" onclick="getElementById('mainContentBox').innerHTML = ''" >Clear</button>
```
Comment (Bsharma4) [inline index.html:29]: @MarshallOkafor This is were Clear button is programed and we don't have to write a separate function about same.

**Review by Bsharma4 — COMMENTED** [review]
Kindly review & merge the same.

File: index.html (hunk 4 above)
Comment (MarshallOkafor) [reply index.html:29]: @Bsharma4 Great! I inadvertently skipped the label of the button. Thanks for letting me know. I have also seen in it when I updated and run the code on my system.

Well done guys!

**Pull request merged**